lib/flipper/railtie.rb

```diff
@@ -5,7 +5,8 @@ class Railtie < Rails::Railtie
         env_key: "flipper",
         memoize: true,
         preload: true,
-        instrumenter: ActiveSupport::Notifications
+        instrumenter: ActiveSupport::Notifications,
+        logging: true
```

Comment:
Truly minor thing: memoize, preload, log maybe are more consistent. Or memoizing, preloading, logging, instrumenting. Perhaps `log: true` or `logger: true` would be more consistent? 

I also think I'd default to `false`. Then people can easily turn it on if they like. 

Last thought: this would only work if instrumenter is set to ActiveSupport::Notifications so I wonder if those settings need to be tied together somehow. You have any preference or thoughts on that?

Reply:
> Truly minor thing: memoize, preload, log maybe are more consistent. Or memoizing, preloading, logging, instrumenting. Perhaps `log: true` or `logger: true` would be more consistent?

hah, I had the same argument in my head. My eyes liked that `logging` is the same length as all the other options, but my frontal lobe thought that was a pretty ridiculous reason to chose a name for something. Apparently my eyes won that argument.

`logger` makes it seem like you can set the logger instance. I guess we could support that too.

> I also think I'd default to `false`. Then people can easily turn it on if they like.

I personally really like seeing flipper in the logs. I get that it can get noisy in apps with a ton of flags, but in smaller apps, it's really nice.

> Last thought: this would only work if instrumenter is set to ActiveSupport::Notifications so I wonder if those settings need to be tied together somehow. You have any preference or thoughts on that?

Ah ok, we can add a check to make sure instrumenter it's AS::Notifications

Reply:
> hah, I had the same argument in my head.

great minds...

> I personally really like seeing flipper in the logs.

I can go either way so you can make the call.